modules/gdscript/gdscript_parser.cpp

```diff
@@ -597,12 +597,34 @@ GDScriptParser::Node *GDScriptParser::_parse_expression(Node *p_parent, bool p_s
 			OperatorNode *op = alloc_node<OperatorNode>();
 			op->op = OperatorNode::OP_CALL;
 
+			//Do a quick Array and Dictionary Check.  Replace if either require no arguments.
+			bool replaced = false;
+
 			if (tokenizer->get_token() == GDScriptTokenizer::TK_BUILT_IN_TYPE) {
+				Variant::Type ct = tokenizer->get_token_type();
+				if (ct == Variant::ARRAY) {
```

Comment:
just check p_parsing_constant argument of parse_expression, if so do not apply this logic

Reply:
Ah, thanks.  Missed that.